packages/app/pages/_app.tsx

```diff
@@ -85,28 +85,26 @@ const App = ({ Component, pageProps, theme }: AppProps & PropsType) => {
                 {theme !== 'edpub' && (
                     <Header user={user} isAuthenticated={isAuthenticated} />
                 )}
-                <Layout>
-                    {canLoadPage ? (
-                        <Component
-                            {...pageProps}
-                            isAuthenticated={isAuthenticated}
-                            theme={theme}
-                            user={user}
-                        />
-                    ) : (
-                        <></>
-                    )}
-
-                    {/* todo: consider making this composable, returning children or login prompt? */}
-                    {router.pathname !== '/installation' && (
-                        <UserAuthentication
-                            isAuthenticated={isAuthenticated}
-                            onUserUpdate={setUser}
-                            user={user}
-                        />
-                    )}
-                    <Toast />
-                </Layout>
+                {canLoadPage ? (
```

Comment:
Our dashboard layout needed the escape from this `<Layout>`. In several pages you'll see `<Layout>` added. Without using the `app` folder, our version of Next.js doesn't support layout files per level of nesting, so we just add it in manually to the other pages.